src/main/kotlin/org/rust/ide/refactoring/extractEnumVariant/RsExtractEnumVariantProcessor.kt

```diff
@@ -54,9 +54,16 @@ class RsExtractEnumVariantProcessor(
         val parameters = filterTypeParameters(element.typeReferences, enum.typeParameterList)
         val typeParametersText = parameters.format()
         val whereClause = buildWhereClause(enum.whereClause, parameters)
+        val attributes = findTransitiveAttributes(ctx, TRANSITIVE_ATTRIBUTES)
```

Comment:
Maybe it's better to pass `enum` instead of `ctx` here?
Sorry for a lot of single comments